### ISSUE TITLE
Get acamd database started

### ISSUE THREAD

**Issue by chazshapiro** [CaltechOpticalObservatories/NGPS#145]
Make sure db exists and try writing to it

**Comment by chazshapiro** [comment]
telemetry.acamd exists:

```
    Field          Type     Null                          Comment                         
-------------- ------------ ---- ---------------------------------------------------------
      datetime  datetime(3)   NO                                   YYYY-MM-DD hh:mm:ss.sss
          name varchar(128)  YES                                               Target name
        obs_id          int  YES                                            observation ID
            RA  varchar(32)  YES                                    Target RA (hh:mm:ss.s)
          DECL  varchar(32)  YES                                   Target DEC (dd:mm:ss.s)
         RAtel  varchar(32)  YES                                       TCS RA (hh:mm:ss.s)
       DECLtel  varchar(32)  YES                                      TCS DEC (dd:mm:ss.s)
       airmass        float  YES                                               TCS airmass
            az        float  YES                                               TCS azimuth
      casangle        float  YES                                 TCS Cassegrain ring angle
        filter  varchar(16)  YES                                  Filter selection on ACAM
     pointmode   varchar(8)  YES                           Pointing target at slit or ACAM
         focus        float  YES                                        TCS focus position
      RAoffset        float  YES                                         PT input (arcsec)
    DECLoffset        float  YES                                         PT input (arcsec)
      acquired      tinyint  YES TRUE means ACAM is in guiding mode after acquiring target
         flags  varchar(32)  YES                                                        --
       RAsolve        float  YES                 Astrometry solution for ACAM center (deg)
      DECsolve        float  YES                 Astrometry solution for ACAM center (deg)
      ANGsolve        float  YES         Astrometry solution for ACAM position angle (deg)
    rms_arcsec        float  YES                  RMS residuals of solver WCS fit (arcsec)
        result   varchar(8)  YES           Result of astrometry solver (GOOD, NOISY, etc.)
       matches          int  YES                     Number of matches sources from solver
        seeing        float  YES                      median seeing in ACAM field (arcsec)
    seeing_zen        float  YES                    seeing extrapolated to zenith (arcsec)
    seeing_std        float  YES                      STDDEV seeing in ACAM field (arcsec)
    background        float  YES            median background in ACAM field (mag/argsec^2)
background_std        float  YES            STDDEV background in ACAM field (mag/argsec^2)
    extinction        float  YES            median extinction in ACAM field (mag/argsec^2)
extinction_std        float  YES            STDDEV extinction in ACAM field (mag/argsec^2)
```

**Comment by astronomerdave** [comment]
Database connection and writing has been implemented, though not all fields are being written yet. I've added a checkmark `x` beside those columns in your table above that I have implemented.

**Comment by astronomerdave** [comment]
@chazshapiro I have all but these 4:
```
           flags  varchar(32)  YES                                                        --
      rms_arcsec        float  YES                  RMS residuals of solver WCS fit (arcsec)
         matches          int  YES                     Number of matches sources from solver
      seeing_std        float  YES                      STDDEV seeing in ACAM field (arcsec)
  extinction_std        float  YES            STDDEV extinction in ACAM field (mag/argsec^2)
```
  and I'm not sure where to get them

**Comment by astronomerdave** [comment]
implemented in commit 2397b6462f4dbb567226c1ab0e431c8f5236b892
